terraform/modules/marklogic/marklogic_cf_template.yml

```diff
@@ -483,6 +483,9 @@ Resources:
         - Key: marklogic:stack:id
           Value: !Ref 'AWS::StackId'
           PropagateAtLaunch: 'true'
+        - Key: Name
+          Value: MarkLogic-ASG-1
```

Comment:
This doesn't apply to existing instances, for now I've added it manually on test